IdentityCore/src/webview/embeddedWebview/MSIDAADOAuthEmbeddedWebviewController.m

```diff
@@ -57,6 +57,34 @@ - (id)initWithStartURL:(NSURL *)startURL
                            context:context];
 }
 
+#if MSAL_JS_AUTOMATION
```

Comment:
Same here, rather than duplicating code, can we have a writable property instead?

Reply:
Cleaned all the initializers and used property.